utils/atomic-storybook/.storybook/map-props-to-args.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable node/no-extraneous-import */
```

Comment:
Atomic/bueno are not installed as direct dependency in storybook, they are copied at build time (if I remember that part right).

Hence why we need to disable the tool here

Reply:
ah, actually, I think this is a leftover from back when I tested adding `@coveo/atomic` as a dependency. I may have had to remove it to avoid npm hoisting dependencies, but you might be able to re-add `@coveo/atomic` as a dependency to `@coveo/atomic-storybook`.